libs/remix-ui/run-tab/src/lib/reducers/runTab.ts

```diff
@@ -135,8 +135,7 @@ export const runTabInitialState: RunTabState = {
     }, {
       id: 'web3-mode',
       dataId: 'settingsWeb3Mode',
-      title: `Execution environment connects to node at localhost (or via IPC if available), transactions will be sent to the network and can cause loss of money or worse!
-      If this page is served via https and you access your node via http, it might not work. In this case, try cloning the repository and serving it via http.`,
+      title: `Execution environment connects to an external node. For security, only connect to trusted networks. If Remix is served via https and your node is accessed via http, it might not work. In this case, try cloning the repository and serving it via http.`,
```

Comment:
this is very long and difficult to read at once. `In this case, try cloning the repository and serving it via http.` can be removed

Reply:
I agree this is veering to more info than is appropriate for a tooltip - but it could still be helpful if someone is patient enough to re-activate the tooltip once it fades out.  Or if this is made into a custom tooltip - maybe it should have a longer live in the active state.  So I'd prefer to leave this edit for now.